tools/dgeni/templates/entry-point.template.html

```diff
@@ -36,11 +36,11 @@ <h2>
     API reference for Angular {$ doc.packageDisplayName $} {$ doc.displayName $}
   </h2>
 
-  <p class="docs-api-module-import">
-    <code>
-      import {{$ doc.primaryExportName $}} from '{$ doc.moduleImportPath $}';
-    </code>
-  </p>
+  <div
+    docs-api-module-import
+    exportName="{{$ doc.primaryExportName $}}"
```

Comment:
Isn't this going to include the start/end curly braces in the attribute? It might be better to leave the raw name in the attribute and have the consumer format it appropriately.

Reply:
Yes it is. I was trying to find the raw value property, but was unlucky about it. Will try to fix this asap

Reply:
fixed